components/Comments/CommentForm.vue

```diff
@@ -76,22 +87,38 @@
       }
     },
     methods: {
+      cancel (form) {
+        if (this.isCommentFormOfContribution) {
+          form.content = ''
+        } else {
+          if (this.replyComment)
+            this.$parent.closeCommentForm()
+        }
+      },
       reply (comment) {
         if (!comment) {
           return
         }
-        try {
-          this.$refs.editor.$refs.editorMentions.insertMention(0, comment.user)
-          this.$scrollTo(this.$refs.editor.$el, 500)
-        } catch (err) {}
+        const comments = this.$parent.$el.classList.contains('comments') ? this.$parent : this.$parent.$parent.$parent
+        if (!this.isExecuted && !comments.hasSubmitted && this.$el.children[1].parentElement.id != 'comment-form') {
+          this.isExecuted = true;
+          this.$nextTick(function () {
+            this.$refs.editor.$refs.editorMentions.insertMention(0, comment.user)
+            this.$scrollTo(this.$el.children[1], 500)
+            setTimeout(() => { this.isExecuted = false; }, 700)
+          })
+        }
       },
       async submitComment () {
+        const comments = this.$parent.$el.classList.contains('comments') ? this.$parent : this.$parent.$parent.$parent
```

Comment:
Please do not try to access parents, especially not over multiple levels. Use component props instead (see one way data flow with props in vue documentation)

Reply:
Is it also possible to listen to an event, which is fired three elements down?

Reply:
Therefore you could use the comment vuex store

Reply:
I used an action, mutation and getter in the comments store instead of the above construct